### DISCUSSION TITLE
❓ VAD modelQuestions / Help / Support

### DISCUSSION THREAD

**Discussion by hyttyfj211** [snakers4/silero-vad#120] [Q&A]
## ❓ Questions and Help

We have a [wiki](https://github.com/snakers4/silero-models/wiki) available for our users. Please make sure you have checked it out first.
What method does your VDA model use to train? CNN or what?

**Comment by hyttyfj211** [comment]
Is there an article about VAD model?

**Reply by snakers4** [reply]
Hi,

https://github.com/snakers4/silero-vad/discussions/107

Stay tuned for a large VAD update soon
